HomeTask_9.py

```diff
@@ -1,10 +1,24 @@
+"""HomeTask_9.
+
+write script that will read "input.txt" file and find there all characters from english alphabet
+collect these characters and their positions in file
+after info collected this info as text write to "output.txt" file in such format
+####################
+<first found letter> -> pos1
+<next_letter> -> pos2
+<next_letter -> pos3
+etc
+#######################
+"""
 import re
-with open('input.txt', 'r') as inp:
+with open('input.txt', 'r') as inp:  # open the file for reading
     data = inp.read()
-    regexp = re.findall(r'[a-zA-Z]', data)
-    with open('output.txt', 'w') as out:
-        for i in regexp:
+    regexp = re.findall(r'[a-zA-Z]', data)  # find the all characters from english alphabet
+    with open('output.txt', 'w') as out:  # open the file for writing
```

Comment:
Better prepare data here ( before write) and print them . Then below block with read file is not required.
Read /Write file operations are time consuming and depends on system events ( possible errors) - thus best practices - minimize write calls. In our case you able to prepare whole string and write it once